packages/aws-cdk-migration/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "aws-cdk-migration",
+  "version": "0.0.0",
```

Comment:
How are you thinking about publishing this? Is this published on the v2 branch using v2 versioning or v1?

Reply:
perhaps `aws-cdk-migration@2.0` is for migration to v2 and `aws-cdk-migration@3.0` is for migration to v3?

Reply:
Whatever versioning we pick is going to be confusing. It should be broken off of the CDK versioning as soon as we have that facility, because it's not actually related. Right now the v2 rewrite tool is called `rewrite-imports-v2`, and we sure as hell aren't going to make v3 need a `rewrite-imports-v3` :).

Reply:
You're right. I find it weird to have one package that is a general 'migration' tool. Might have been more pragmatic to have separate packages for each MV upgrade. They will likely have nothing to do with each other.

Nevertheless, I don't expect you to change this.

I suppose your current choice is to publish this on master, so `1.x.y` (at least in the beginning).